keyboards/nibiria/stream15/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
```

Comment:
Via should not be enabled in the default keymap, and instead needs to be it's own keymap. 

```suggestion
```